xdsl/dialects/omp.py

```diff
@@ -32,18 +39,67 @@
     base,
     irdl_attr_definition,
     irdl_op_definition,
+    operand_def,
     opt_operand_def,
     opt_prop_def,
+    prop_def,
     region_def,
+    result_def,
     traits_def,
     var_operand_def,
 )
 from xdsl.parser import AttrParser
 from xdsl.printer import Printer
-from xdsl.traits import IsolatedFromAbove, IsTerminator, NoTerminator
+from xdsl.traits import IsolatedFromAbove, IsTerminator, NoMemoryEffect, NoTerminator
 from xdsl.utils.exceptions import VerifyException
 
 
+class OpenMPOffloadMappingFlags(IntFlag):
```

Comment:
this is unused

Reply:
It would be used when setting or testing the value of `map_type` in `omp.map.info`. `map_type` is a bit set, so the enum is not actually used in the definition of the operation.